# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxWindowsPlatform.h
# Commit: 228200dcaf93
# Full Hash: 228200dcaf93bb3f10e338157325ef6227e8a0a5
# Author: Kelsey Gilbert <kelsey.gilbert@mozilla.com>
# Date: 2023-02-15 18:16:35
# Regressor Bug: 1799258
# File Overlap Count: 1
# Description:
#   Bug 1799258 - Do color-management on Windows+DComp via IDCompositionFilterEffects. r=sotaro
#   
#   + Add gfx.color_management.rec709_gamma_as_srgb:true. :'(
#   
#   In particular, rec709(16/255) -> srgb(31/255). Even though it's
# ==============================================================================

diff -r b25110652394 -r 228200dcaf93 gfx/thebes/gfxWindowsPlatform.h
--- a/gfx/thebes/gfxWindowsPlatform.h	Wed Feb 15 08:29:41 2023 +0000
+++ b/gfx/thebes/gfxWindowsPlatform.h	Wed Feb 15 08:29:41 2023 +0000
@@ -200,7 +200,13 @@
 
  protected:
   bool AccelerateLayersByDefault() override { return true; }
+
   nsTArray<uint8_t> GetPlatformCMSOutputProfileData() override;
+
+ public:
+  static nsTArray<uint8_t> GetPlatformCMSOutputProfileData_Impl();
+
+ protected:
   void GetPlatformDisplayInfo(mozilla::widget::InfoObject& aObj) override;
 
   void ImportGPUDeviceData(const mozilla::gfx::GPUDeviceData& aData) override;